user-guide/Advanced_Functionality/Swarming/SwarmingFAQ.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: SwarmingFAQ
+---
+
+# Frequently Asked Questions
+
+## Can I Swarm X/Y/Z?
+
+Swarming will gradually become available for more and more functionality in DataMiner.
+As of DataMiner 10.5.1, only basic elements are supported.
+
+## Does the ID of an element change when it is Swarmed?
+
+No, the identifier of the element stays the same.
+
+Each element also stores where it is hosted, that field is updated when Swarmed.
+
+## Can I block an element from Swarming?
+
+Not in the initial release but is expected to be available in DataMiner 10.5.5/10.6.0.
+
+## What does the **DataMinerID** mean of an element that Swarmed?
+
+The meaning of a **DataMinerID** has slightly changed in a Swarming DMS.
+
+It used to be that every element has an **ElementID** that uniquely identifies it on a DataMiner agent.
+However, once clusters of agents were created, these were no longer unique in the whole DMS, so the **DataMinerID** was added to the element identifier, which now consists of 2 number often shown as **DataMinerID/ElementID**.
+
+![What does the DataMinerID of an element mean](~/user-guide/images/Swarming_FAQ_DataMinerID.png)
+
+To find the agent that was hosting the element, that same **DataMinerID** was used.
+However this no longer works when Swarming the element.
+To keep the identifier unique in the DMS this **DataMinerID** cannot change.
+
+To see the agent hosting the element in DataMiner Cube, you can click a parent view and scroll to the right until you see the **HOST ID** and **DATAMINER** column for the element.
+
+Example below (with columns reordered for convenience):
+
+![Hosting Agent in Cube](~/user-guide/images/Swarming_FAQ_HostingAgentCube.png)
+
+To figure out the agent hosting the element in code, you can use the **HostingAgentID** property instead.
+
+## What happens with my Elements folder on disk?
+
+The configuration of an element used to be stored in an XML file on disk.
```

Comment:
Maybe add a reference to "C:\Skyline DataMiner\Elements", or just link here: https://docs.dataminer.services/user-guide/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/Elements1.html#elements

Reply:
Done